x-pack/plugins/apm/public/components/shared/AgentIcon/get_agent_icon.ts

```diff
@@ -37,6 +39,11 @@ const agentIcons: { [key: string]: string } = {
   rum: rumJsIcon,
 };
 
+const darkAgentIcons: { [key: string]: string } = {
+  php: darkPhpIcon,
```

Comment:
if you add `...agentIcons,` above this line then all icons will work in dark mode.